docs/how-to/h-deploy-azure.md

```diff
@@ -0,0 +1,242 @@
+# How to deploy on Azure
+
+[Azure](https://azure.com/) is the cloud computing platform developed by Microsoft. It has management, access and development of applications and services to individuals, companies, and governments through its global infrastructure. Access the Azure web console at [portal.azure.com](https://portal.azure.com/).
+
+## Summary
+
+* [Install Azure and Juju tooling](#install-client-environment)
+  * [Authenticate](#authenticate)
+* [Bootstrap Juju controller on Azure](#bootstrap-juju-controller-on-azure)
+* [Deploy charms](#deploy-charms)
+* [Clean up](#clean-up)
+
+---
+
+## Install Client Environment
+
+> **WARNING**: The current limitations:
+> * Only supported starting Juju 3.6 (currently edge)
```

Comment:
??

Reply:
yes, the auth process is not working as below otherwise and it is a bit more clunky, requiring people to create an App on the Azure portal and use those secrets. The latter process is also not the recommended way forward. So right now we should use Juju 3.6 if we want to deploy on Azure